elki/src/main/java/de/lmu/ifi/dbs/elki/utils/containers/MwpIndex.java

```diff
@@ -0,0 +1,18 @@
+package de.lmu.ifi.dbs.elki.utils.containers;
+
+/**
+ * Structure to hold return values in index creation for McdeMwpDependenceMeausre
+ */
+
+public class MwpIndex extends RankStruct{
+    public double adjusted;
```

Comment:
public fields (or ideally, any field) should come with a JavaDoc documentation. It may be appropriate to make this a static inner class of McdeMwpDependenceMeasure, if this is not useful independently of it.

Reply:
RankStruct and MwpIndex are now inner classes of the new MCDETest and MWPTest accordingly